docs/settings.md

```diff
@@ -9,13 +9,13 @@ of dealing with the different levels and exposes easy to use getters and setters
 ## Levels
 
 Granular Settings rely on a series of known levels in order to use the correct value for the scenario. These levels, in 
-order of prioirty, are:
+order of priority, are:
 * `device` - The current user's device
 * `room-device` - The current user's device, but only when in a specific room
 * `room-account` - The current user's account, but only when in a specific room
 * `account` - The current user's account
 * `room` - A specific room (setting for all members of the room)
-* `config` - Values are defined by the `settingDefaults` key (usually) in `config.json`
+* `config` - Values are defined by the `settingDefaults` key (usually) in `config.tson`
```

Comment:
What's a `tson`? (Assuming find / replace error...)

Reply:
find & replace all got the better of me ._.